docker-compose/grafana/provisioning/dashboards/nats-surveyor-dashboard.json

```
@@ -776,6 +776,900 @@
         "x": 0,
         "y": 4
       },
+      "id": 248,
+      "panels": [],
+      "title": "Jetstream Status",
```

sorry updating JetStream casing...

```suggestion
      "title": "JetStream Status",
```

---

updated